carapace-ui/src/main/webapp/src/App.vue

```diff
@@ -1,113 +1,116 @@
+            headerCollapsed="CA"
+            :elements="[
+        {
+            label: 'Backends',
+            icoName: 'server',
+            href: '/',
+        },
+        {
+            label: 'Directors',
+            icoName: 'crosshairs',
+            href: '/directors',
+        },
+        {
+            label: 'Actions',
+            icoName: 'bolt',
+            href: '/actions',
+        },
+        {
+            label: 'Routes',
+            icoName: 'map-signs',
+            href: '/routes',
+        },
+        {
+            label: 'Listeners',
+            icoName: 'door-open',
+            href: '/listeners',
+        },
+        {
+            label: 'Headers',
+            icoName: 'heading',
+            href: '/headers',
+        },
+        {
+            label: 'Filters',
+            icoName: 'filter',
+            href: '/requestfilters',
+        },
+        {
+            label: 'Certificates',
+            icoName: 'file-signature',
+            href: '/certificates',
+        },
+            {
+            label: 'Cache',
+            icoName: 'archive',
+            href: '/cache',
+        },
+        {
+            label: 'Users',
+            icoName: 'users',
+            href: '/users',
+        },
+        {
+            label: 'Metrics',
+            icoName: 'chart-bar',
+            href: '/metrics',
+        },
+        {
+            label: 'Peers',
+            icoName: 'network-wired',
+            href: '/peers',
+        },
+        {
+            label: 'Configuration',
+            icoName: 'sliders-h',
+            href: '/configuration',
+        },
 
-                    <li>
-                    <router-link to="/configuration">Configuration</router-link>
-                    </li>
-                </ul>
-            </div>
-            <div id="page-content-wrapper">
-                <div class="container-fluid">
-                    <router-view></router-view>
-                </div>
-            </div>
-        </div>
+    ]"
+        />
+        <router-view class="main flex-grow-1 p-2" />
     </div>
 </template>
 
 <script>
-    import { doGet } from './mockserver'
-    export default {
-        name: 'app',
-        data: function () {
-            return {
-                peerId: ""
-            }
-        },
-        created: function () {
-            var url = "/api/cluster/localpeer"
-            var d = this
-            doGet(url, response => {
-                d.peerId = response.id
-                document.title += ' (' + d.peerId + ')'
-            })
-        },
+import { doGet } from "./mockserver";
+import Sidebar from "./components/Sidebar.vue";
+
+export default {
+    name: "app",
+    components: {
+        sidebar: Sidebar
+    },
+    data() {
+        return {
+            peerId: ""
+        };
+    },
+    created() {
+        var url = "/api/cluster/localpeer";
+        var self = this;
+        doGet(url, data => {
```

Comment:
Using lamba functions "this" keyword has the same value as outside of the function, so in this case self === this